src/miral/basic_window_manager.cpp

```diff
@@ -511,7 +511,17 @@ void miral::BasicWindowManager::force_close(Window const& window)
 
 auto miral::BasicWindowManager::active_window() const -> Window
 {
-    return mru_active_windows.top();
+    auto const focussed_surface = focus_controller->focused_surface();
```

Comment:
Small typo, it would be focused_surface in English :)

Reply:
Well, this code is copied from another code in the same file. Also, "focussed" is kind of a valid spelling in British English (and Canonical is a UK-based company so... :) ).

https://en.wiktionary.org/wiki/focussed

Reply:
Its just odd that in the same line of code you have 2 spellings... If you want to keep it though I wont block it ofc.

Reply:
Oh, didn't notice that there're 2 spellings in the same line. Then I'll fix it.